airbyte-cdk/python/setup.py

```diff
@@ -15,7 +15,7 @@
 
 setup(
     name="airbyte-cdk",
-    version="0.9.5",
+    version="0.9.6",
```

Comment:
since this is a new backwards compatible feature not a bug, we should increment the minor version